dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/presenter/impl/BasePresenterImpl.java

```diff
@@ -46,8 +46,6 @@ protected void bindBaseElements() {
         view.getHeaderViewAboutLink().setOnAction(arg0 -> view.showAboutPopup());
         
         view.getHeaderViewHelpLink().setOnAction(arg0 -> view.showHelpPopup());
-        
-        view.getCancelLink().setOnAction(arg0 -> getController().goToPreviousPage());

```

Comment:
Wouldn't it have been cleaner to just check if cancel link is not null here, instead of moving it to each of the individual presenters?